src/udev/cdrom_id/cdrom_id.c

```diff
@@ -987,14 +1248,22 @@ int main(int argc, char *argv[]) {
                 printf("ID_CDROM_MEDIA_BD=1\n");
         if (cd_media_bd_r)
                 printf("ID_CDROM_MEDIA_BD_R=1\n");
+        if (cd_media_bd_r_srm)
+                printf("ID_CDROM_MEDIA_BD_R_SRM=1\n");
+        if (cd_media_bd_r_rrm)
+                printf("ID_CDROM_MEDIA_BD_R_RRM=1\n");
         if (cd_media_bd_re)
                 printf("ID_CDROM_MEDIA_BD_RE=1\n");
         if (cd_media_hddvd)
                 printf("ID_CDROM_MEDIA_HDDVD=1\n");
         if (cd_media_hddvd_r)
                 printf("ID_CDROM_MEDIA_HDDVD_R=1\n");
+        if (cd_media_hddvd_r_dl)
+                printf("ID_CDROM_MEDIA_HDDVD_R_DL=1\n");
         if (cd_media_hddvd_rw)
                 printf("ID_CDROM_MEDIA_HDDVD_RW=1\n");
+        if (cd_media_hddvd_rw_dl)
+                printf("ID_CDROM_MEDIA_HDDVD_RW_DL=1\n");
```

Comment:
given how large this already was, and how much larger this will get now, can we please rework this to be a table? an array of structs mapping the values to properties to set?